### DISCUSSION TITLE
[Vue-cli] urql error can't resolve graphql when installing

### DISCUSSION THREAD

**Discussion by ccsv** [urql-graphql/urql#2796] [Q&A]
### Describe the bug

Installed urql/vue 1.02 using the vue cli on a windows machine and got this error

```
error  in ./node_modules/@urql/core/dist/ddbb86ae.mjs

Module not found: Error: Can't resolve 'graphql' in 'C:\data\...'
```

It works if you use npm directly:

`npm install --save @urql/vue graphql`

### Reproduction

see above

### Urql version

urql 3.03

### Validations

- [X] I can confirm that this is a bug report, and not a feature request, RFC, question, or discussion, for which [GitHub Discussions](https://github.com/FormidableLabs/urql/discussions) should be used
- [X] Read the [docs](https://formidable.com/open-source/urql/docs/).
- [X] Follow our [Code of Conduct](https://github.com/FormidableLabs/urql/blob/main/CODE_OF_CONDUCT.md)

**Comment by JoviDeCroock** [comment]
I mean, the instructions do clearly say to install `graphql` alongside `@urql/vue` so not really sure why it would do that automatically with the vue-cli. Converting this to a discussion as well as it's more of a question rather than a bug.
